docs/source/cpp/compute.rst

```diff
@@ -1056,7 +1056,8 @@ Temporal component extraction
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 These functions extract datetime components (year, month, day, etc) from timestamp type.
-Note: this is currently not supported for timestamps with timezone information.
+Note: supports timestamps with timezone information and will return localized timestamp
+components.
 
 +--------------------+------------+-------------------+---------------+----------------------------+-------+
 | Function name      | Arity      | Input types       | Output type   | Options class              | Notes |
```

Comment:
By the way, one question: the table below says "temporal" for the input types, but only timestamp inputs are supported, right?

Reply:
Yes. I've changed the columns to say Timestamp to reflect this.